### ISSUE TITLE
Incorrect warning: "`@meta` block has an unsupported keyword argument: CurrentModule" since v1.2.0

### ISSUE THREAD

**Issue by tkemmer** [JuliaDocs/Documenter.jl#2364]
Documenter v1.2.0 throws a warning when encountering the `CurrentModule` keyword in a `@meta` block,
although the keyword still seem to exist and work as before (and is still shown in the [documentation](https://documenter.juliadocs.org/stable/man/guide/#Adding-Some-Docstrings)):
```
┌ Warning: In src/index.md:1-3: `@meta` block has an unsupported keyword argument: CurrentModule
└ @ Documenter ~/.julia/packages/Documenter/ifiMq/src/expander_pipeline.jl:298
```
The warning occurs for each document containing something like this:
```
```@meta
CurrentModule = MyModule
...
```

I think this might be a typo in

https://github.com/JuliaDocs/Documenter.jl/blob/master/src/expander_pipeline.jl#L296

The code checks against `:Currentmodule` instead of `:CurrentModule` before generating the warning.

**Comment by fredrikekre** [comment]
Do you want to make a PR?

**Comment by tkemmer** [comment]
Sure! I'll prepare one.

**Comment by fredrikekre** [comment]
Fixed by #2365
